app/models/chargeback.rb

```diff
@@ -44,6 +44,37 @@ def self.refresh_dynamic_metric_columns
     set_columns_hash(dynamic_rate_columns)
   end
 
+  def self.report_settings
+    @@report_settings_cash ||= begin
```

Comment:
Should all the chargeback reports really be using the same currency?

(this is a per-system config, not per user?)

(If the currency is per-user, the cashing sounds surprising.)

Reply:
I think you want `report_settings_cash` to be `report_settings_cache`. But, since this is dealing with currency, maybe `cash` is the way to go, lol